{{ cookiecutter.github_repo_name }}/.github/workflows/tests-on-pr.yml

```diff
@@ -0,0 +1,14 @@
+name: Tests on PR
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  tests-on-pr:
+    uses: Billingegroup/release-scripts/.github/workflows/_tests-on-pr.yml@v0
+    secrets:
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Leave CODECOV_TOKEN for now since it's a required parameter.. Created an issue https://github.com/Billingegroup/scikit-package-system/issues/4